Fix failing OCSP stapling test of `IO::Socket:SSL` extension after the www.chksum.de server disabled OCSP support.
See: https://github.com/noxxi/p5-io-socket-ssl/issues/169
https://github.com/noxxi/p5-io-socket-ssl/commit/58ad1d1672516d246f7536961c6ab320ffde1b8a
This fix is included only from the 2.090 release of IO-Socket-SSL.

--- a/t/external/fingerprint.pl
+++ b/t/external/fingerprint.pl
@@ -8,10 +8,10 @@
 # --- BEGIN-FINGERPRINTS ----
 my $fingerprints= [
   {
-    _ => 'this should give us OCSP stapling',
+    _ => 'this should give us OCSP stapling - before LetsEncrypt had disabled OCSP support',
     fingerprint => 'sha1$pub$39d64bbaea90c6035e25ff990ba4ce565350bac5',
     host => 'www.chksum.de',
-    ocsp => {
+    _disabled_ocsp => {
               staple => 1
             },
     port => 443
